src/rng/backends/rocrand/mkl_rng_rocrand_wrappers.cpp

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (C) 2022 Heidelberg University, Engineering Mathematics and Computing Lab (EMCL) 
+ * and Computing Centre (URZ)
+ * cuRAND back-end Copyright (c) 2021, The Regents of the University of
```

Comment:
rocRAND

Reply:
Shouldn't we consistently use cuRAND in the old license everywhere?

Reply:
> Shouldn't we consistently use cuRAND in the old license everywhere?

sorry my mistake.